client/admin_test.go

```diff
@@ -19,102 +18,66 @@ import (
 )
 
 func TestCollectLogs(t *testing.T) {
-	type ticker struct {
-		tick <-chan time.Time
-		tock <-chan time.Time
-	}
-
-	newTicker := func() ticker {
-		return ticker{
-			tick: time.After(50 * time.Millisecond),
-			tock: time.After(60 * time.Millisecond),
-		}
-	}
-
 	t.Run("access logs", func(t *testing.T) {
-		clock := newTicker()
+		i := 0
 		receiver := func() (*responsev1.ListAuditLogEntriesResponse, error) {
-			select {
-			case <-clock.tick:
+			for ; i < 1; i++ {
```

Comment:
Unless I am missing something, this loop doesn't do anything here. Removing it would not affect anything at all.

Reply:
ah yeah, it left there.